homeassistant/components/roborock/strings.json

```diff
@@ -136,7 +136,11 @@
           "washing_the_mop": "Washing the mop",
           "going_to_wash_the_mop": "Going to wash the mop",
           "charging_complete": "Charging complete",
-          "device_offline": "Device offline"
+          "device_offline": "Device offline",
```

Comment:
There should be a map defined in the integration for the status sensor enum options. The library should not define those completely on its own.

If there's an update to the library that changes the status codes an update to the integration code should be required before new state is returned. Only defining the translation keys isn't enough.